content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -27,6 +27,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
 * Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
+    * For example: `sudo swapoff -a` (and make sure this change is permanent. Swap may be set in files like `/etc/fstab`, `systemd.swap` depending how it was configured on your system)
```

Comment:
```suggestion
  For example, on some Linux systems, you can do `sudo swapoff -a`.
  To make this change persistent across reboots, wwap may be set in files like
  `/etc/fstab`, `systemd.swap` depending how it is configured on your system.
```

Reply:
Updated in https://github.com/kubernetes/website/pull/40654/commits/b2b8299841f45e5da35fa474a019906fdda056fd